swagger.yaml

```diff
@@ -767,7 +767,7 @@ paths:
               schema:
                   $ref: "#/components/schemas/Error"
 
-  "/severa/{severaUserId}/resourceAllocations":
+  "/severa/users/{severaUserId}/resourceAllocations":
```

Comment:
Having userId as a path param rather than an optional query param will mean we always need userId to request the resourceAllocation. DO you think this is fine? I don't think we need to worry too much about admin functionality for now so maybe it is fine, just wanted to mention it.